stringsubstitutor/src/test/java/org/jdbi/v3/stringsubstitutor/TestStringSubstitutorSqlParser.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jdbi.v3.stringsubstitutor;
+
+import org.jdbi.v3.core.statement.ParsedSql;
+import org.jdbi.v3.core.statement.StatementContext;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(MockitoJUnitRunner.class)
```

Comment:
Prefer `@Rule MockitoRule mockitoRule = MockitoJUnit.rule()` over the test runner.

Reply:
Cool, I didn't know you could do that. Makes tests nicely extensible, lets you combine mockito with other stuff that usually needs a Runner. Thanks for telling